apple/RNCWebView.m

```diff
@@ -426,6 +426,8 @@ - (void)didMoveToWindow
     _webView.scrollView.pagingEnabled = _pagingEnabled;
     //For UIRefreshControl to work correctly, the bounces should always be true
     _webView.scrollView.bounces = _pullToRefreshEnabled || _bounces;
+    _webView.scrollView.alwaysBounceHorizontal = _pullToRefreshEnabled || bounces;
+    _webView.scrollView.alwaysBounceVertical = _pullToRefreshEnabled || bounces;
```

Comment:
Does this compile? Shouldn't it be using `_bounces` instead of `bounces`?

Reply:
Yeah sure I didn't mean to open a PR yet, it was supposed to be in draft. I just fixed it 👌